### ISSUE TITLE
Custom routes not available! error message

### ISSUE THREAD

**Issue by sceiler** [robvanderleek/create-issue-branch#539]
Hi @robvanderleek,

I'm looking to use your Action but I get the above mentioned error message when it runs. One run is here https://github.com/saucelabs/saucerest-java/runs/4455967886?check_suite_focus=true

```
Run robvanderleek/create-issue-branch@main
/usr/bin/docker run --name a6825e5bac7bdf36c46fc93323b7e5b573295_a2c5f8 --label 6a6825 --workdir /github/workspace --rm -e GITHUB_TOKEN -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/saucerest-java/saucerest-java":"/github/workspace" 6a6825:e5bac7bdf36c46fc93323b7e5b573295
App was loaded
Custom routes not available!
Skipping Sentry.io setup
Skipping branch creation for issue: Remove obsolete maintainer from POM
```

My workflow yml is this:
```
name: Create Issue Branch
on:
    issues:
        types: [assigned]
    issue_comment:
        types: [created]

jobs:
    create_issue_branch_job:
        runs-on: ubuntu-latest
        steps:
        - name: Create Issue Branch
          id: Create_Issue_Branch
          uses: robvanderleek/create-issue-branch@main
          env:
            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        - name: Echo branch name
          run: echo ${{ steps.Create_Issue_Branch.outputs.branchName }}
```

With these overrides:

```
gitSafeReplacementChar: '-'
branchName: '${issue.number}-${issue.title}'
defaultBranch: 'dev'
branches:
  - label: enhancement
    prefix: feature/
    name: dev
    skip: false
  - label: bug
    name: staging
    prefix: bug/
    skip: false
  - label: '*'
    skip: true
```

Any idea why this happens?

**Comment by robvanderleek** [comment]
Hi @sceiler 

Thanks for your feedback.

The message is not something to worry about. Custom routes are only available when Create Issue Branch is running as a GitHub App, not as a GitHub Action.

Does the action works for you as expected?

I can suppress this error if CIB is running as a GitHub Action.

KR, Rob

**Comment by sceiler** [comment]
Hi @robvanderleek 
I don't know what happened or changed but now it works. Yesterday when I assigned an issue to myself the Action ran successfully but no branch was created. I just tried again ant the branch was created as expected. 🤷‍♂️ 
I think it would be a good idea to suppress the warning if it is irrelevant when running as an Action though.

Because it is working now I will just close the issue. Thanks for your help!

**Comment by robvanderleek** [comment]
Hey @sceiler 

I've merged PR #540 that suppresses the warning you encountered, thanks for reporting 😄 

I did notice the Action output you provided in this issue contained `Skipping branch creation for issue: Remove obsolete maintainer from POM` so maybe one of your configuration options (like not having a bug or enhancement label) prevented a branch creation the first time?

Best regards, Rob
